rest_framework_supertest/shortcuts/images.py

```diff
@@ -0,0 +1,32 @@
+import io
+import uuid
+from typing import Optional
+
+from django.core.files import File
+
+
+def image(
+    fake: object,
+    width: int = 256,
+    height: int = 256,
+    image_format: str = 'png',
+    file_name: Optional[str] = None,
+) -> File:
+    """
+    Generate a image to store it to django file field.
+
+    Args:
+        fake: The `Faker` instance.
+        width: The image width.
+        height: The image height.
+        image_format: The image_format to store. See Pillow image file formats.
+        file_name: The file name to store into `django.core.files.File` instance.
+    """
+    data = fake.image(size=(width, height), image_format=image_format)
```

Comment:
As we are using original faker image method, we can remove the "customizated" image provider and run tests to grant the integrity of the application.